### PR TITLE
Add spoil time to Mustard

### PR DESCRIPTION
**Pull request by KurzedMetal**
#### Summary
SUMMARY: Bugfixes "Add spoil date to Mustard"

#### Purpose of change
Mustard isn't currently spoiling.

#### Describe the solution
Add spoil_in to JSON, I used the same spoil time as Ketchup (I did a quick search and they seem to have similar spoilage)

### CODE REVIEW

**Comment by Maleclypse** [comment]
Is there currently a way to reliably create mustard after all the mustard has expired as there is with ketchup or mayonnaise?

**Comment by KurzedMetal** [comment]
It seems it doesn't exist, I'm guessing because mustard seeds don't exist either, someone needs to add a lot of stuff to be able to craft them.

And the only recipe that use mustard that can't be replaced by something else is "deviled kidney"

**Comment by Rivet-the-Zombie** [comment]
Mustard doesn't generally spoil. It might dry out after a long enough time, but it never really seems to go bad.

**Comment by KurzedMetal** [comment]
I googled, and most resources mentioned it had a shelve life similar to ketchup, and ketchup has a spoil time.

So we either need to remove spoiling from ketchup too or add it to mustard to make it consistent.

Googling about it, I found that Mustard & Ketchup spoils in 2 years if the bottle is not opened, and about 1-2 months when opened. Ketchup currently has 3 week spoil time in-game, so I assumed it was counted as it was opened.

**Comment by kevingranade** [comment]
Yea we're assuming uncontained, to represent opened bottles sitting around,
we need to have additional support for that.

**Comment by CountAlex** [comment]
I think I can add mustard flowers, seeds and recipe for homemade mustard (it's pretty simple anyway). The only thing is I don't know if mustard plant is present in New England. I'll check this and, if there's no objections, will add them in new PR.

**Comment by Maleclypse** [comment]
I’m not sure if it will natively spread but people definitely grow it in home gardens up here.

**Comment by jeremyshannon** [comment]
> 1-2 months when opened in pantry

That seems really low for a spoilage date, on more than one occasion I've had an opened bottle of mustard sitting in the back of a pantry for nigh on a year, and while it got darker and drier, it came out tasting a little strong but not bad. Vinegar is a pretty good preservative.
 Now if that 1-2 months is a "best by" date, that's different. It definitely tastes better fresh.

**Comment by KurzedMetal** [comment]
> Yea we're assuming uncontained, to represent opened bottles sitting around, we need to have additional support for that.

What do you think of the idea of adding a two new condiment containers for ketchup/mustard, one with sealed+preserves flags and another unsealed, so the ketchup/mustard never spoils until opened for the first time? This would simulate their very long life when unopened.

Then I could make an item group with a distribution having a small percentage of spawning sealed and use that in every place currently spawning mustard.

**Comment by Maleclypse** [comment]
As someone with a bunch of unopened mustard in my house that I have to stir up a little when I get to it, I approve that plan.

**Comment by KurzedMetal** [comment]
I commited the change I suggested about the new condiment container. Please someone review if that's a good idea or not.

File: data/json/items/containers.json [hunk 1]
```diff
@@ -219,6 +219,42 @@
     "seals": true,
     "watertight": true
   },
+  {
+    "id": "condiment_bottle_sealed",
+    "type": "CONTAINER",
+    "category": "container",
+    "name": { "str": "condiment bottle" },
+    "description": "An inverted plastic bottle for condiments. Still sealed from factory, preserves content from rot until opened.",
```
Comment (kevingranade) [inline data/json/items/containers.json:227]:
```suggestion
    "description": "An inverted plastic bottle for condiments.  Still sealed from factory, preserves content from rot until opened.",
```

**Pull request merged**